vlib/v/fmt/tests/select_keep.vv

```diff
@@ -108,8 +108,7 @@ fn test_select_blocks() {
 		else {
 			h = 2
 		}
-	}
-	{
+	} {
```

Comment:
that looks odd 🤔

Reply:
the whole file looks confusing to me :/

Reply:
I'm open to better ways to format this

Reply:
is it
```
}
{
```
or
```
}{
```
?

Did you revert to the behaviour i introduced?

Reply:
Yeah, as it was not needed anymore